src/ucs/debug/debug.c

```diff
@@ -1094,7 +1095,7 @@ static inline void ucs_debug_save_original_sighandler(int signum,
     khiter_t hash_it;
     int hash_extra_status;
 
-    pthread_spin_lock(&ucs_kh_lock);
+    ucs_spin_lock(&ucs_kh_lock);
```

Comment:
`ucs_spinlock_acquire`

Reply:
I can't find `ucs_spinlock_acquire ` in the latest master (c303724fd7bc85ef9609f92c1b5cd575b6a1e1f4)
What am I missing?